lib/koine/repository/persistence/adapter.rb

```diff
@@ -2,31 +2,31 @@ module Koine
   module Repository
     module Persistence
       class Adapter
-        def exists?(criterias)
+        def exists?(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `exists?(*)` if you want the method to accept any arguments but don't care about them.